app/src/assets/plugins/app.search.json

```diff
@@ -104,36 +104,6 @@
             { "query": "-TYPE:'lnk:link'" },
             { "query": "-PNAME:'0/wiki'" }
           ],
-          "facetQueries": {
-            "label": "SEARCH.CATEGORIES.MODIFIED_DATE",
-            "expanded": true,
-            "queries": [
-              {
-                "label": "SEARCH.FACET_QUERIES.TODAY",
```

Comment:
we can probably remove these translations from translation files?

Reply:
Good point, removed